test/remote/gateways/remote_orbital_test.rb

```diff
@@ -118,7 +118,7 @@ def setup
     ]
   end
 
-  def test_successful_purchase
+  def test_successful_purchase_1
```

Comment:
why rename this test?

Reply:
oh It's my bad, 🤦🏻‍♂️  doing testing, becuase there is another `test_successful_purchase` function on the line 1238, for `TandemOrbitalTests`. thanks